packages/devtools/lib/src/memory/memory.dart

```diff
@@ -332,7 +332,7 @@ class MemoryScreen extends Screen with SetStateMixin {
   }
 
   void _resetHistory() {
-    history..clazz('history-navigation', removeOthers: true)..clazz('hidden');
+    history.attribute('hidden');
```

Comment:
Much simpler :-)

Reply:
ah, sorry, I didn’t see the `.history-navigation` css